packages/schema/src/language-server/zmodel-linker.ts

```diff
@@ -278,9 +278,11 @@ export class ZModelLinker extends DefaultLinker {
                 const model = getContainingModel(node);
 
                 if (model) {
-                    const userModel = getAllDeclarationsFromImports(this.langiumDocuments(), model).find(
-                        (d) => isDataModel(d) && d.name === 'User'
-                    );
+                    const dataModel = this.getContainingDataModel(node);
```

Comment:
I think we need to find data model with `@@auth` attribute from all data model declarations, but not just the container of the current `auth()` invocation. Just like how we used to find the model named "User" in the code below this.

Reply:
ok, let me try that